### DISCUSSION TITLE
Nomor sering ketendang (bukan banned)

### DISCUSSION THREAD

**Discussion by RakaiSeto** [aldinokemal/go-whatsapp-web-multidevice#198] [Q&A]
Jadi kek sering, saya abis scanned, tar abis ngirim beberapa message malah ketendang (tapi ga kebanned), cuka sessionnya kelepas, kenapa ya

**Comment by aldinokemal** [comment]
running di server cloud apa lokal bro?

**Reply by kickymaulana** [reply]
loh ini yang buat orang lokal indo toh?

**Reply by aldinokemal** [reply]
yoi gan

**Reply by kickymaulana** [reply]
owalah aku kira orang luar negeri. udah 3 tahun pakai ini gak tau yang buat orang indo. semangat
